server/auth.js

```diff
@@ -0,0 +1,9 @@
+import { generateKeyPairSync } from 'crypto';
+
+const { privateKey } = generateKeyPairSync('rsa', {
+  modulusLength: 4096,
+  privateKeyEncoding: { type: 'pkcs8', format: 'pem' },
+  publicKeyEncoding: { type: 'pkcs1', format: 'pem' },
+});
```

Comment:
Generates the token automatically. I left the `JWT` env var there so the current behaviour keeps working, just removes a setup step for someone launching for the first time.